<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Spectrum</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/gui/menu.png);</string>
   </property>
   <widget class="QGraphicsView" name="gwDisplay">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>800.000000000000000</width>
      <height>600.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResumeGame">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>Resume Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>400</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNewGame">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChooseLevel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>Choose Level</string>
    </property>
   </widget>
   <widget class="QFrame" name="frLevelTree">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>751</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba(30, 30, 30, 0.99);
background-image: url(:/gui/lvltree.png); </string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pbLevel1">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>100</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>001</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLevel2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>002</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLevelDemo">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>100</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbHideLevelPanel">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>Hide Level Tree</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLevel3">
     <property name="geometry">
      <rect>
       <x>423</x>
       <y>100</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>003</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLevel4">
     <property name="geometry">
      <rect>
       <x>207</x>
       <y>207</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>004</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLevel5">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>207</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>005</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLevel6">
     <property name="geometry">
      <rect>
       <x>493</x>
       <y>207</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>006</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLevel7">
     <property name="geometry">
      <rect>
       <x>422</x>
       <y>312</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>007</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLevel8">
     <property name="geometry">
      <rect>
       <x>277</x>
       <y>312</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>008</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLevel9">
     <property name="geometry">
      <rect>
       <x>132</x>
       <y>312</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>009</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLevel10">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>424</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>010</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLevel11">
     <property name="geometry">
      <rect>
       <x>349</x>
       <y>424</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>011</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLevel12">
     <property name="geometry">
      <rect>
       <x>494</x>
       <y>424</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>012</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLevel13">
     <property name="geometry">
      <rect>
       <x>565</x>
       <y>312</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>013</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLevel14">
     <property name="geometry">
      <rect>
       <x>638</x>
       <y>424</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;</string>
     </property>
     <property name="text">
      <string>014</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnLoadGame">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>Load Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveGame">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>Save Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHelp">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border: 1px solid white;</string>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QFrame" name="frHelp">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>500</width>
      <height>375</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba(30, 30, 30, 0.99);
background-image: url(:/gui/help.png); </string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnHideHelp">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
border: 1px solid white;
background: none;
</string>
     </property>
     <property name="text">
      <string>Hide</string>
     </property>
    </widget>
   </widget>
   <zorder>gwDisplay</zorder>
   <zorder>btnResumeGame</zorder>
   <zorder>btnExit</zorder>
   <zorder>btnNewGame</zorder>
   <zorder>btnChooseLevel</zorder>
   <zorder>btnLoadGame</zorder>
   <zorder>btnSaveGame</zorder>
   <zorder>btnHelp</zorder>
   <zorder>frLevelTree</zorder>
   <zorder>frHelp</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>on_pbLevel_clicked()</slot>
 </slots>
</ui>
